src/components/SendActivityModal/SendActivityModal.stories.tsx

```
@@ -372,3 +372,122 @@ export const CompleteHierarchy: Story = () => {
+    itens: [
+      { id: 'turma-unica', name: '9º Ano A - Manhã', yearId: 'serie-unica' },
+    ],
+    selectedIds: [],
+  },
+  {
+    key: 'alunos',
+    label: 'Alunos',
+    dependsOn: ['turma'],
+    filteredBy: [{ key: 'turma', internalField: 'classId' }],
+    itens: [
+      { id: 's1', name: 'Ana Carolina Silva', classId: 'turma-unica', studentId: 's1', userInstitutionId: 'ui1' },
```

ESLint

[error] Replace `·id:·'s1',·name:·'Ana·Carolina·Silva',·classId:·'turma-unica',·studentId:·'s1',·userInstitutionId:·'ui1'` with `⏎········id:·'s1',⏎········name:·'Ana·Carolina·Silva',⏎········classId:·'turma-unica',⏎········studentId:·'s1',⏎········userInstitutionId:·'ui1',⏎·····`

(prettier/prettier)